src/_optPlug.node/xmlValidator.ts

```diff
@@ -27,7 +27,7 @@ export default opWrapper<Functionality>('XmlValidator', [
   /* eslint-disable @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-require-imports
      -- needed */
 
-  ['libxmljs2', () => require('./__xmlValidators/libxmljs2').default]
+  ['libxml2-wasm', () => require('./__xmlValidators/libxml2-wasm').default]
```

Comment:
do not modify the line, but add a new one below

Reply:
Added the line instead of replacing